js/coinsbit.js

```diff
@@ -0,0 +1,1099 @@
+            'amount': amount,
+            'cost': cost,
+            'fee': fee,
+        }, market);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        if (type !== 'limit') {
+            throw new InvalidOrder (this.id + ' createOrder supports limit orders only');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+            'side': side,
+            'amount': this.amountToPrecision (symbol, amount),
+            'price': this.priceToPrecision (symbol, price),
+        };
+        const response = await this.privatePostOrderNew (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //       orderId: '8629354782',
+        //       market: 'ETH_USDT',
+        //       price: '4000',
+        //       side: 'buy',
+        //       type: 'limit',
+        //       timestamp: '1640212275.928',
+        //       dealMoney: '7.98856357',
+        //       dealStock: '0.002',
+        //       amount: '0.002',
+        //       takerFee: '0.002',
+        //       makerFee: '0.002',
+        //       left: '0',
+        //       dealFee: '0'
+        //     },
+        //     code: '200'
+        //   }
+        const data = this.safeValue (response, 'result', {});
+        return this.parseOrder (data, market);
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new InvalidOrder (this.id + ' createOrder supports limit orders only');
+        }
+        await this.loadMarkets ();
+        const market = this.safeMarket (undefined, symbol);
+        const request = {
+            'market': market['id'],
+        };
+        request['orderId'] = id;
+        const response = await this.privatePostOrderCancel (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //       orderId: '8631519189',
+        //       market: 'ETH_USDT',
+        //       price: '3100',
+        //       side: 'buy',
+        //       type: 'limit',
+        //       timestamp: '1640254497.745',
+        //       dealMoney: '0',
+        //       dealStock: '0',
+        //       amount: '0.002',
+        //       takerFee: '0.002',
+        //       makerFee: '0.002',
+        //       left: '0.002',
+        //       dealFee: '0'
+        //     },
+        //     code: '200'
+        //  }
+        const data = this.safeValue (response, 'result');
+        return this.parseOrder (data);
+    }
+
+    async fetchOpenOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' fetchOrders requires symbol parameter');
```

Comment:
If they require a symbol, we should throw early, before `loadMarkets()`.
If possible, please edit the error message, so that it always reads uniformly across all exchanges and all methods:
```JavaScript
throw new ArgumentsRequired (this.id + ' fetchOrders() requires a symbol argument');
```